src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -447,6 +447,21 @@ PARAM_DEFINE_FLOAT(MPC_DEC_HOR_SLOW, 5.0f);
  */
 PARAM_DEFINE_FLOAT(MPC_ACC_HOR_FLOW, 0.5f);
 
+/**
+ * Maximum height above ground when reliant on optical flow.
+ * The height setpoint will be limited to be no greater than this value when the navigation system
+ * is completely reliant on optical flow data and the height above ground estimate is valid as indicated
+ * by the local_position.dist_bottom_valid message being true.
+ *
+ * @unit m
+ * @min 1.0
+ * @max 25.0
+ * @increment 0.5
+ * @decimal 1
+ * @group Multicopter Position Control
+ */
+PARAM_DEFINE_FLOAT(MPC_MAX_FLOW_HGT, 3.0f);
```

Comment:
Can we have the flow-related control params names consistently?

e.g `MPC_MAX_HGT_FLOW` to match with `MPC_ACC_HOR_FLOW`